lib/common/options.c

```diff
@@ -255,7 +255,7 @@ static const pcmk__cluster_option_t cluster_options[] = {
         NULL,
     },
     {
-        PCMK_OPT_STONITH_TIMEOUT, NULL, "time", NULL,
+        PCMK_OPT_STONITH_TIMEOUT, NULL, PCMK_VALUE_TIMEOUT, NULL,
         "60s", pcmk__valid_interval_spec,
```

Comment:
Anything using interval_spec is a duration (eventually all timeouts will be durations and we can drop timeout as a type, see T166)

Reply:
whoops, probably goofed this based on stonith-watchdog-timeout